products/amazon-rds-mysql.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Amazon RDS for MySQL
+category: db
+iconSlug: mysql
+permalink: /rds-mysql
+releasePolicyLink: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Concepts.VersionMgmt.html
+
+releases:
+-   releaseCycle: "8.0"
+    releaseDate: 2018-10-23
+    eol: 2026-04-01
+    latest: "8.0.32"
+    latestReleaseDate: 2023-02-07
+
+-   releaseCycle: "5.7"
+    releaseDate: 2016-02-22
+    eol: 2023-10-01
+    latest: "5.7.41"
+    latestReleaseDate: 2023-02-07
+
+-   releaseCycle: "5.6"
+    releaseDate: 2013-07-01
+    eol: 2022-03-01
+    latest: "unknown"
+
+---
+
+> [Amazon RDS for MySQL](https://aws.amazon.com/rds/mysql) is a PaaS offering from Amazon for
+> creating managed MySQL Community Edition databases. RDS makes it easier to set up, operate, and
+> scale MySQL deployments on AWS cloud.
+
+**MySQL recommends that all users run the latest available minor release for whatever major version
```

Comment:
Is it a MySQL recommendation or an AWS recommandation ?

Reply:
I looked into this more and MySQL actually doesn't recommend always upgrading to latest because there is no simple reversal.

RDS is managed MySQL. So, while it does makes things easy by doing a backup of the database and makes sure it's stable before applying upgrades. There are 2 major risks involved.
- upgrades can go past maintenance windows and cause unexpected downtime
- upgrades themselves cause downtime (even with Multi-availability zone features enabled) and without proper setup the most stable thing for most production cases is to disable automatic minor upgrades.

Some of this stuff is necessary for my job to know but I agree with your comment. This seems outside the scope of endoflife to comment on this.

I will simply state there are risks and that people should research them on their own.